README.md

```diff
@@ -11,7 +11,7 @@ http://cosmos-snap.staketab.com/ - Main page os snapshots.
 http://cosmos-snap.staketab.com/ixo - IXO (Mainnet)  
 http://cosmos-snap.staketab.com/medibloc - Medibloc (Mainnet)  
 http://cosmos-snap.staketab.com/stargaze - Stargaze (Mainnet)  
-http://cosmos-snap.staketab.com/sifchain - Sifchain (Betanet)  
+http://cosmos-snap.staketab.com/sifchain - Sifchain (Betanet) 
 
```

Comment:
Add please link to snap in Mainnet section (https://mercury-nodes.net/kichain-snaps/)

Reply:
Done

Reply:
Done